docs/guides/react-native.md

```diff
@@ -9,32 +9,188 @@ a host device.
 This can be difficult when you want your test suite to run with typical Continuous Integration servers
 such as Travis.
 
-A pure JS mock of React Native exists and can solve this problem in the majority of use cases.
+To use enzyme to test React Native, you currently need to configure an adapter, and load an emulated DOM.
 
-To install it, run:
+## Configuring an Adapter
 
-```bash
-npm i --save-dev react-native-mock
+While a React Native adapter is [in discussion](https://github.com/airbnb/enzyme/issues/1436),
+a standard adapter may be used, such as "enzyme-adapter-react-16":
+
+```
+import Adapter from "enzyme-adapter-react-16";
+
+Enzyme.configure({ adapter: new Adapter() });
+```
+
+## Loading an emulated DOM with JSDOM
+
+To use enzyme's `deep` until a React Native adapter exists, an emulated DOM must be loaded.
```

Comment:
what is "deep"? Do you mean `mount`?

Reply:
Yes mount, good catch, I'll update that.

Reply:
This has been updated to `mount`